### PR TITLE
[ext-docs] SidePanel API related docs

### PR DESCRIPTION
**Pull request by AmySteam**
Other docs related to the SidePanel API launch PR #6178 

- Manifest key
- Permissions table
- Architecture
- Design UI
- What's new announcement

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ tags:
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about many of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).)
 
+### Chrome 114: New SidePanel API for Manifest V3 {: #m114-side-panel}
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/extensions/whatsnew/index.md:19]: @jpmedley Good idea I'll tweak this to add the samples. I need to wait to merge this until all other side panel related items are approved.

[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/docs/extensions/whatsnew/index.md]: FYI, there's nothing going out today, and probably nothing until next week. There's no need to remove this. Just make sure it's on top.

**Review by oliverdunk — APPROVED** [review]
LGTM with nit 👍

File: site/en/docs/extensions/mv3/architecture-overview/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: "layouts/doc-post.njk"
 title: "Architecture overview"
 seoTitle: "Chrome Extensions architecture overview"
 date: 2012-09-18
-updated: 2023-01-10
+updated: 2023-05-24
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/architecture-overview/index.md:6]:
```suggestion
updated: 2023-05-30
```

File: site/en/docs/extensions/mv3/user_interface/index.md [hunk 3]
```diff
@@ -3,11 +3,10 @@ layout: "layouts/doc-post.njk"
 title: "Design the user interface"
 seoTitle: "Chrome Extensions: Design the user interface"
 date: 2018-03-16
-updated: 2022-04-27
+updated: 2023-05-24
```
Comment (AmySteam) [inline site/en/docs/extensions/mv3/user_interface/index.md:6]:
```suggestion
updated: 2023-05-30
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ layout: 'layouts/doc-post.njk'
 title: What's new in Chrome extensions
 description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
 date: 2021-02-25
-updated: 2023-05-17
+updated: 2023-05-24
```
Comment (AmySteam) [inline site/en/docs/extensions/whatsnew/index.md:6]:
```suggestion
updated: 2023-05-30
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 5]
```diff
@@ -15,13 +15,26 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
-### New Samples: WASM in extensions
+### Chrome 114: New Side Panel API {: #side-panel-launch }
+
+<p class="color-secondary-text type--caption">Posted on <time>May 24, 2023</time></p>
```
Comment (AmySteam) [inline site/en/docs/extensions/whatsnew/index.md:20]:
```suggestion
<p class="color-secondary-text type--caption">Posted on <time>May 30, 2023</time></p>
```

**Review by AmySteam — COMMENTED** [review]
Update date

**Review by IanStanion-google — APPROVED** [review]
LGTM!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**